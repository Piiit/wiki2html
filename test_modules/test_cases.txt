Before everything...
**test** 123 
**bold2** //italic//
== head = er 2 ===
//italic**bold**//
**bold//italic//bold**
...this is the end!%
= this is a header (level 1) =
lala.

a ==
END

* list item 1
* **bold list item 2** something //italic//
* list item

x={{x}}.
EOF
