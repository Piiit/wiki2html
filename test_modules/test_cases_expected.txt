Before everything...
<b>test</b> 123 
<b>bold2</b> <i>italic</i>
<h2> head = er 2 </h2>
<i>italic<b>bold</b></i>
<b>bold<i>italic</i>bold</b>
...this is the end!%
<h1> this is a header (level 1) </h1>
lala.

a ==
END

<ul>
<li> list item 1</li>
<li> <b>bold list item 2</b> something <i>italic</i></li>
<li> list item</li>
</ul>

x=. 
EOF
